lib/components/Layout/styles/layout.css

```diff
@@ -5,6 +5,17 @@
   left: 0;
 }
 
+.container, .container a, .container p {
```

Comment:
I'm a fan of new lines for these. What do you think?

```css
.container, 
.container a,
.container p {

}
```

Reply:
@avkvirtru can we get the stylelinter formatter to do this for us?